tests/terraform/runner/test_plan_runner.py

```diff
@@ -276,7 +276,7 @@ def test_plan_runner_with_empty_vpc_connection(self):
         self.assertEqual(report.get_exit_code({'soft_fail': False, 'soft_fail_checks': [], 'soft_fail_threshold': None, 'hard_fail_checks': [], 'hard_fail_threshold': None}), 1)
         self.assertEqual(report.get_exit_code({'soft_fail': True, 'soft_fail_checks': [], 'soft_fail_threshold': None, 'hard_fail_checks': [], 'hard_fail_threshold': None}), 0)
 
-        self.assertEqual(report.get_summary()["failed"], 105)
+        self.assertEqual(report.get_summary()["failed"], 106)
```

Comment:
The test tfplan has a mysql RDS instance without `require_secure_transport`

https://github.com/bridgecrewio/checkov/blob/379d66af024f83dd1ef32fc44d9ea6dfdd7f01ec/tests/terraform/runner/resources/plan/tfplan.json#L143